test/update_test.rb

```diff
@@ -157,6 +163,8 @@ def default_SetDesktopPattern_stubs
     let(:os_backup_release_content) { File.new("#{DATA_DIR}/etc/tw-os-release").read }
 
     before do
+      allow(Yast::Update.log).to receive(:info)
```

Comment:
You do not need to mock the logging calls. When running as non-root, which is usual for tests, the log is saved to `$HOME/.y2log`. And actually you should not mock it, the logging might be useful if a test fails, you might find some details there.